titan-options.php

```diff
@@ -489,44 +495,115 @@ function regala_create_options() {
 	/**
 	 *   Sections
 	 */
-    
-    $havanaSections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Sections', 'regala' ),
-        'panel' => __( 'Theme Options & Colors', 'regala' ),
-    ) );
-    
-    $sections = $titan->createThemeCustomizerSection( array(
-        'name' => __( 'Section 1', 'regala' ),
-        'panel' => __( 'Sections', 'regala' ),
-    ) );
-    
-    $sections->createOption( array(
-        'name' => __( 'Layout', 'regala' ),
-        'id' => 'section_1',
-        'type' => 'radio-image',
-        'options' => array(
-            'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
-            'layout2' => get_template_directory_uri() . '/images/header-image-default1.jpg',
-            'layout3' => get_template_directory_uri() . '/images/header-image-default2-thumb.jpg',
-            'layout4' => get_template_directory_uri() . '/images/header-image-default2.jpg',
-            'layout5' => get_template_directory_uri() . '/images/logo.png',
-            'layout6' => get_template_directory_uri() . '/images/site-icon.png',
-         ),
-    ) );    
-  
-    
-	
+         
+    for ( $sectionIndex = 1; $sectionIndex <= 5; $sectionIndex++ ) {
+        for ( $boxIndex = 1; $boxIndex <= 4; $boxIndex++ ) {
+             $sectionLayout = $titan->createThemeCustomizerSection( array(
+                 'name' => __( 'Section ' . $sectionIndex . ' Layout ', 'havana' ),
+                 'panel' =>  __('Section ' . $sectionIndex, 'havana'),
```

Comment:
`__()` functions should not have variables inside them.

Turn this:
```__('Section ' . $sectionIndex, 'havana')```

Into this:
```sprintf( __( 'Section %d', 'havana' ), $sectionIndex )```

Research:
* http://codex.wordpress.org/I18n_for_WordPress_Developers
* http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/
* http://php.net/sprintf